internal/expr/typeinfo.go

```diff
@@ -39,10 +39,6 @@ type structField struct {
 
 	// The tag assosiated with this field
 	tag string
-
-	// OmitEmpty is true when "omitempty" is
-	// a property of the field's "db" tag.
-	omitEmpty bool
```

Comment:
Let's please keep this. See the note regarding the three cases mentioned in the prior review. This is one of them. It's just not what is being handled right now.

Reply:
Added back.